lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -83,6 +83,10 @@ class PostgreSQLGetLastArchivedWALError(Exception):
     """Exception raised when retrieving last archived WAL fails."""
 
 
+class PostgreSQLGetCurrentTimelineError(Exception):
+    """Exception raised when retrieving last archived WAL fails."""
```

Comment:
Shouldn't the docstring be `Exception raised when retrieving current archived WAL fails.`?

Reply:
Fixed :)

Reply:
that was quick! thanks